frontend/src/tests/Navbar.test.js

```diff
@@ -1,14 +1,22 @@
-import React from 'react'
+import React from "react";
 
-import {fireEvent, render, screen} from '@testing-library/react'
+import { fireEvent, render, screen } from "@testing-library/react";
 
-import Navbar from '../components/Navbar'
+import Navbar from "../components/Navbar";
+import App from "../App";
 
-test('Navbar is rendered correctly', ()=>{
-    render(<Navbar />);
-    const navlinkHomeText = screen.getByText('Home')
-    const navlinkMatchText = screen.getByText('Match')
-    
-    expect(navlinkHomeText).toBeInTheDocument()
-    expect(navlinkMatchText).toBeInTheDocument()
-})
+test("Navbar is rendered correctly", () => {
+  render(<App />);
+  const navlinkHomeText = screen.getByText("Home");
+  const navlinkMatchText = screen.getByText("Match");
+
+  expect(navlinkHomeText).toBeInTheDocument();
+  expect(navlinkMatchText).toBeInTheDocument();
+});
+
+test("Navbar navigates to appropriate views", () => {
+  render(<App />);
+  fireEvent.click(screen.getByText("Home"));
+  const element = screen.getByText("Landing Page Component");
```

Comment:
When I ran the test this test failed because of this line. I think its because this text doesnt exist anymore.

Reply:
Tests were changed the match the new components' text and passed in the latest commit on my branch.